apps/web/src/lib/core/component/LexicalMarkdown/styles.css

```
@@ -53,56 +74,62 @@
 
 .md-list.md-bullet > li::before {
   content: " ";
-  width: 0.4em;
-  height: 0.4em;
+  --md-bullet-size: 0.4em;
+  width: var(--md-bullet-size);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)